index.rst

```diff
@@ -77,19 +85,39 @@ categorized as:
 Tag Fragments
 -------------
 
-In general, tags can be postfixed with an optional Cycle number or a
-``rest`` set of one-or-more underscore-separated fields.  The Cycle is
-currently unique to T&S RSP instances, and specifies the version of XML
-used in this image.  It will have the form ``c<digits>.<digits>`` where
-the first group of digits is the cycle number and the second group of
-digits is the build iteration for that cycle, e.g. ``c0019.001``.
+Primary
+^^^^^^^
+The primary version of a tag is derived from (and in most cases identical to) the EUPS tag of the underlying DM stack included in the image.
```

Comment:
I think this is technically the Pipelines stack, which is more specific than DM.